MapboxCoreNavigation/RouteController.swift

```diff
@@ -566,6 +568,15 @@ extension RouteController: CLLocationManagerDelegate {
                 RouteControllerNotificationUserInfoKey.locationKey: self.location!, //guaranteed value
                 RouteControllerNotificationUserInfoKey.rawLocationKey: location //raw
                 ])
+            
+            if let currentIntersection = routeProgress.currentLegProgress.currentStepProgress.currentIntersection,
+                let classes = currentIntersection.outletRoadClasses {
+                if classes.contains(.tunnel) {
+                    startTunnelAnimation(for: manager, routeProgress: routeProgress, distanceTraveled: distanceTraveled)
```

Comment:
Why is `distanceTraveled` being passed in here?

Reply:
When location simulation is needed in special cases such as `dead reckoning`, the `distanceTraveled ` on the given route determines where the `SimulatedLocationManager` needs to begin the simulation.